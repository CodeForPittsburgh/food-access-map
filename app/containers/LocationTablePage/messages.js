/*
 * LocationTablePage Messages
 *
 * This contains all the text for the LocationTablePage container.
 */
import { defineMessages } from 'react-intl';

export const scope = 'app.containers.LocationTablePage';

export default defineMessages({
  header: {
    id: `${scope}.header`,
    defaultMessage: 'Locations',
  },
});
