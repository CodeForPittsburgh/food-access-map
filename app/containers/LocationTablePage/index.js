/*
 * LocationTablePage
 *
 * This is a sortable searchable table of the available locations
 *
 */

import React from 'react';
import Grid from '@material-ui/core/Grid';
import { FormattedMessage } from 'react-intl';
import { Link } from 'react-router-dom';
import LocationTable from '../../components/LocationTable';
import messages from './messages';

export default function LocationTablePage() {
  return (
    <Grid
      container
      direction="column"
      justify="space-around"
      alignItems="center"
    >
      <Link to="/">I go back home! Why am I not a reusable nav?!</Link>
      <h1 className="headline">
        <FormattedMessage {...messages.header} />
      </h1>
      <LocationTable />
    </Grid>
  );
}
