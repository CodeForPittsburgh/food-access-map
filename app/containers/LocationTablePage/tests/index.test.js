import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import LocationTablePage from '../index';

const renderer = new ShallowRenderer();

describe('<LocationTablePage />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<LocationTablePage />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
