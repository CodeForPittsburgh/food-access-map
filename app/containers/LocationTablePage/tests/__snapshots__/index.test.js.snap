// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LocationTablePage /> should render and match the snapshot 1`] = `
<ForwardRef(WithStyles)
  alignItems="center"
  container={true}
  direction="column"
  justify="space-around"
>
  <Link
    to="/"
  >
    I go back home! Why am I not a reusable nav?!
  </Link>
  <h1
    className="headline"
  >
    <FormattedMessage
      defaultMessage="Locations"
      id="app.containers.LocationTablePage.header"
      values={Object {}}
    />
  </h1>
  <Memo(LocationTable) />
</ForwardRef(WithStyles)>
`;
